test/jdk/sun/net/www/http/KeepAliveStream/KeepAliveStreamCloseWithWrongContentLength.java

```diff
@@ -102,12 +140,10 @@ public static void main (String[] args) throws Exception {
                 }
             }
             is.close();
-        } catch (IOException e) {
-            return;
-        } catch (NullPointerException e) {
-            throw new RuntimeException (e);
-        } finally {
-            serversocket.close();
         }
+        catch (Exception e) {
+            e.printStackTrace();
+        }
+
```

Comment:
Why catch exception? That will hide test failures. Let exceptions percolate out of main instead.